angular/core/components/group_page/group_help_card/group_help_card.coffee

```diff
@@ -3,11 +3,14 @@ angular.module('loomioApp').directive 'groupHelpCard', ->
   restrict: 'E'
   templateUrl: 'generated/components/group_page/group_help_card/group_help_card.html'
   replace: true
-  controller: ($scope, CurrentUser, AppConfig, UserHelpService) ->
+  controller: ($scope, $sce, CurrentUser, AppConfig, UserHelpService) ->
     $scope.showVideo = AppConfig.loadVideos
 
     $scope.helpLink = ->
       UserHelpService.helpLink()
 
+    $scope.helpVideo = ->
+      $sce.trustAsResourceUrl(UserHelpService.helpVideo())
```

Comment:
Does it not like importing $sce into the service? I think I've had issues doing that with stuff like $location in the past. It's not a showstopper by any means.